system-addon/lib/HighlightsFeed.jsm

```diff
@@ -21,11 +21,16 @@ XPCOMUtils.defineLazyModuleGetter(this, "NewTabUtils",
   "resource://gre/modules/NewTabUtils.jsm");
 XPCOMUtils.defineLazyModuleGetter(this, "Screenshots",
   "resource://activity-stream/lib/Screenshots.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "ProfileAge",
+  "resource://gre/modules/ProfileAge.jsm");
 
 const HIGHLIGHTS_MAX_LENGTH = 9;
 const HIGHLIGHTS_UPDATE_TIME = 15 * 60 * 1000; // 15 minutes
 const MANY_EXTRA_LENGTH = HIGHLIGHTS_MAX_LENGTH * 5 + TOP_SITES_SHOWMORE_LENGTH;
 const SECTION_ID = "highlights";
+const BOOKMARKS_THRESHOLD = 4000; // 3 seconds to milliseconds.
```

Comment:
Is this supposed to be 3 or 4 seconds? Did it need to be one or the other for some particular reason?

Reply:
On my profile the difference between profile creation date and the default bookmarks is ~2.5s. Looking at the timestamp they are added sequentially at ~2-3ms apart. It was initially 3 which was enough to block them on my machine, I just decided to increase that time to be more cautious.
That comment needs to be updated.

Reply:
It would seem that a number that would work for most users as opposed to "enough" for a developer machine would be better to make sure the experience isn't "broken" for average users.

Reply:
I've created a follow up issue https://github.com/mozilla/activity-stream/issues/3633.